src/test/java/uk/gov/hmcts/reform/adoption/common/service/SubmissionServiceTest.java

```diff
@@ -27,21 +36,26 @@ class SubmissionServiceTest {
     @InjectMocks
     private SubmissionService submissionService;
 
```

Comment:
Is the test case working as expected?
Since it was commented out thats why raising this query.